tests/v2/validation/pipeline/qase/testcase/testcase.go

```diff
@@ -0,0 +1,25 @@
+package testcase
+
+type GoTestOutput struct {
+	Time    string `json:"Time" yaml:"Time"`
+	Action  string `json:"Action" yaml:"Action"`
+	Package string `json:"Package" yaml:"Package"`
+	Test    string `json:"Test" yaml:"Test"`
+	Output  string `json:"Output" yaml:"Output"`
+	Elapsed string `json:"Elapsed" yaml:"Elapsed"`
+}
+
+type GoTestCase struct {
+	Name       string
+	Status     string
+	StackTrace string
+	Elapsed    string
+}
+
+// func NewGoTestCase(name, status, stackStrace string) GoTestCase {
+// 	return GoTestCase{
+// 		Name:       name,
+// 		Status:     status,
+// 		StackTrace: stackStrace,
+// 	}
+// }
```

Comment:
Probably something from the development stage

Reply:
fixed